docs/language_guide.md

```diff
@@ -522,37 +634,57 @@ struct Rectangle {
 ```
 
 ### Instances
-The `Rectangle` structure definition describes only what a `Rectangle` would look like and what functionality they contain. They don't describe specific rectangles. To do that you create an instance of that structure.
+
+The declaration of a struct only describes what types of variables it contains, what their initial values are, and what functions may be used to modify or access the struct data. To create concrete instances, each with individual data values, an instance has to be created, by calling the initialiser of a struct.
+
+```swift
+<struct-name>(<initialiser-parameter-values>)
+```
+
+Example:
 
 ```swift
 let someRectangle: Rectangle = Rectangle()
 ```
 
-When you create an instance, you initialize that structure with it's initial values - in this case a width and height of 0. This process can also be done manually using the `init` special declaration. You can access the properties of the current structure with the special keyword `self`
+When an instance is created, it is initialised with its initial values – in this case a width and heigth of `0`. This process can also be done manually using an [initialiser](#initialisers). You can access the properties of the current struct with the special keyword `self`.
+
+Example:
 
 ```swift
 struct Rectangle {
   // Same definition as above with:
-  public init(width: Int, height: Int){
+  public init(width: Int, height: Int) {
     self.width = width
     self.height = height
   }
 }
 ```
 
 ```swift
-let bigRectangle = Rectangle(400, 10000)
+let bigRectangle = Rectangle(width: 400, height: 10000)
 ```
 
-### Accessing Properties/Functions
-You can access properties/functions of an instance using _dot syntax_. In dot syntax, you write the property name immediately after the instance name, separated by a period (.), without any spaces.
+### Accessing properties/functions
 
+Properties/functions of a struct instance can be accessed using dot syntax. In dot syntax, the property name is written immediately after the instance name, separated by a period `.`:
+
+```swift
+<struct-instance>.<variable-name>
+<struct-instance>.<constant-name>
+<struct-instance>.<function-name>(<function-parameter-values>)
 ```
-someRectangle.width // evaluates to 0
-someRectangle.area() // evaluates to 0 by calling the function area on the Rectangle instance someRectangle
+
+Examples:
+
+```swift
+bigRectangle.width // 400
+bigRectangle.area() // evaluates to 4000000 by calling the `area` function
 ```
 
-In Flint, you can also access the functions of a struct using this syntax without creating an instance - as long as they don't contain instance properties. (In the future we will support the `static` keyword to do this)
+In Flint, functions of a struct can also be called without creating an instance – as long as they do not use any instance properties.
```

Comment:
These also seem to not be declared with `func`? Might be worth checking if that's a mistake and if not then ensuring that this nuance is pointed out
News to me that we had static functions. In Swift they're declared with a static keyword which makes more sense.

Reply:
That was a typo, but static functions are not tested anywhere and surprise, surprise, they're broken. Removing this.